src/UI/App.jsx

```diff
@@ -321,6 +322,13 @@ export default class App extends React.Component {
 		// We do this after the mount because `createParleyProxy` contains
 		// `setState()` calls, which should not be called before mounting
 		window.parleySettings = this.createParleyProxy(window.parleySettings);
+
+		if(this.state.messengerOpenState === "open")
+			this.showChat();
+		 else if(this.state.messengerOpenState === "minimize")
+			this.hideChat();
+		 else
+			localStorage.setItem("messengerOpenState", "minimize");
```

Comment:
die else zou niet meer werken met de aanpassing in de state (want er is geen `null` value meer).

Misschien is het beter om naar `localStorage.getItem("messengerOpenState")` te kijken ipv `this.state.messengerOpenState`, zo kan je hem aanmaken in de localStorage als die nog niet bestaat en heb je in je state altijd een correcte value staan (niet `null`)

Reply:
Haal ook even de hoofdletter `M` weg van `MessengerOpenState`. We beginnen vars niet met een hoofdletter

Reply:
Onthouden en Done 5220f363fe4df2536c8401bbfd8d39e460fa82e1

Reply:
Resolved, heeft geen hoofdletter meer